apps/cliffordsynthesis_app.cpp

```diff
@@ -0,0 +1,269 @@
+/*
+* This file is part of the MQT QMAP library which is released under the MIT license.
+* See file README.md or go to https://iic.jku.at/eda/research/ibm_qx_mapping/ for more information.
+*/
+
+#include "LogicBlock/LogicBlock.hpp"
+#include "algorithms/RandomCliffordCircuit.hpp"
+#include "cliffordsynthesis/CliffordSynthesizer.hpp"
+#include "utils/logging.hpp"
+
+#include <boost/program_options.hpp>
+#include <iostream>
+#include <locale>
+
+int main(int argc, char** argv) {
```

Comment:
This is more of a general question: is it really practical to maintain this CLI in parallel to the Python interface?
At least from my point of view, the future of advertising the usage of this tool (and qmap as a whole) is via the Python interface. It allows far easier connection to things like qiskit and is more flexible. Furthermore, the CLI tools are not available when installing via pip. At least in my opinion, they are a pretty high burden and poorly maintained.
I am in favor of removing this CLI entirely.